src/app.js

```diff
@@ -56,7 +56,7 @@ const sessionConfig = {
   ...(SESSION_TABLE_NAME && { sessionStore: dynamoDBSessionStore }),
 };
 
-const helmetConfig = require("@govuk-one-login/di-ipv-cri-common-express/src/lib/helmet");
+const helmetConfig = commonExpress.lib.helmet;
```

Comment:
Helmet is still not being exported - https://github.com/govuk-one-login/ipv-cri-common-express/blob/main/src/lib/index.js

Do you need this line?